applications/callflow/src/cf_endpoint.erl

```diff
@@ -265,6 +266,12 @@ merge_attributes([<<"record_call">> = Key|Keys], Account, Endpoint, Owner) ->
     OwnerAttr = get_record_call_properties(Owner),
     Merged = wh_json:merge_recursive([AccountAttr, OwnerAttr, EndpointAttr]),
     merge_attributes(Keys, Account, wh_json:set_value(Key, Merged, Endpoint), Owner);
+merge_attributes([<<"caller_id_format">> = Key|Keys], Account, Endpoint, Owner) ->
```

Comment:
@mark2600 There is an object that exists called caller_id_options (from years ago) that was supposed to have a 'format' property.  I like the use of classifiers so maybe something like:

caller_id_options: {
    format: {
        us_did: {
            regex: "^\\+?1?([0-9]{10})$"
            prefix: "1"
        }
   }
}

thoughts?